components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/dao/CacheBackedLocalClaimDAO.java

```diff
@@ -75,6 +75,21 @@ public void updateLocalClaim(LocalClaim localClaim, int tenantId) throws ClaimMe
         localClaimInvalidationCache.clearCacheEntry(tenantId, tenantId);
     }
 
+    /**
+     * Update attribute claim mappings related to tenant id and domain in bulk.
+     *
+     * @param localClaimList  list of local claims.
```

Comment:
Description should be in sentence case.

Reply:
Addressed with commit 2e3f3c951772d195bcbedb1a7b81f4815b531259 in https://github.com/wso2/carbon-identity-framework/pull/3774 PR